nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -212,6 +212,68 @@
      <literal>hardware.ckb-next.*</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>nscd</literal> now disables all caching of
+      <literal>passwd</literal> and <literal>group</litral> databases by
```

Comment:
Should be `</literal>` the closing tag in group

Reply:
`nix build -f my-nixpkgs/nixos config.system.build.manual --arg configuration {}` to make sure the manual builds :)